index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css"/>
+    <title>Portfolio</title>
+</head>
```

Comment:
[OPTIONAL]

- [ ] Always write your commit messages in the present tense. Here, you've used the present continuous tense. Instead of `adding images of the project`, write `Add images of the project`. Please refer to the lessons on how to keep a professional repository.

Reply:
thanks @george-shammar